### PR TITLE
docs: add docs quickstarts

### PR DESCRIPTION
**Pull request by minaelee**
- add quickstarts for fern definition and openapi
- remove old quickstart
- update docs.yml and other files that link to docs quickstart

### CODE REVIEW

File: fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+<CodeBlocks>
+    <CodeBlock title='Custom subdomain'>
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com
+      custom-domain: docs.YOUR_ORGANIZATION.COM
+    ```
+    </CodeBlock>
+    <CodeBlock title='Custom subpath'>
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com
```
Comment (dannysheridan) [inline fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:145]: This should be YOUR_ORGANIZATION.docs.buildwithfern.com/docs

File: fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+### Step 7: Set up a custom domain
+
+If you wish to use a custom subdomain like `https://docs.YOUR_ORGANIZATION.COM` or a subpath like `https://YOUR_ORGANIZATION.COM/docs`, you can subscribe to the [Starter plan](https://buildwithfern.com/pricing). Once subscribed, update `docs.yml` with the custom domain configuration, replacing YOUR_ORGANIZATION with your own:
+
+<CodeBlocks>
+    <CodeBlock title='Custom subdomain'>
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com
+      custom-domain: docs.YOUR_ORGANIZATION.COM
```
Comment (dannysheridan) [inline fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:140]: How should we indicate that `docs.` is a choice and can be configured to any string?

File: fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+Need help? Email us at [support@buildwithfern.com](mailto:support@buildwithfern.com) or join our [Discord community](https://discord.com/invite/JkkXumPzcG).
+
+## Customer showcase
+
+Your docs can look this good:
+
+- [Flatfile's API Reference](https://reference.flatfile.com/api-reference/events/create-an-event)
+- [Sugeragent's Docs](https://docs.superagent.sh/)
+- [Credal's Docs](https://docs.credal.ai/)
```
Comment (dannysheridan) [inline fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:165]: Can you swap this to:

1. Flagright https://docs.flagright.com/
2. Superagent
3. Flatfile

File: fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-openapi.mdx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+```yaml
+- url: YOUR_ORGANIZATION.docs.buildwithfern.com
+  custom-domain: docs.YOUR_ORGANIZATION.COM
+```
+
+Custom subpath:
+
+    ```yaml
+    - url: YOUR_ORGANIZATION.docs.buildwithfern.com
```
Comment (dannysheridan) [inline fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-openapi.mdx:145]: I believe this needs to be: YOUR_ORGANIZATION.docs.buildwithfern.com/docs

File: fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx (hunk 1 above)
Comment (minaelee) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:145]: Do you mean `YOUR_ORGANIZATION.buildwithfern.com/docs`?
Or still keep the subdomain `docs` before `buildwithfern`?

File: fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx (hunk 2 above)
Comment (minaelee) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:140]: @dannysheridan Same applies for the subpath, I imagine? E.g. the `docs` in 
```YOUR_ORGANIZATION.COM/docs```
could be anything, like `YOUR_ORGANIZATION.COM/guides` or whatever they want?

Comment (dannysheridan) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:140]: exactly.

File: fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx (hunk 1 above)
Comment (dannysheridan) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:145]: I believe it is: YOUR_ORGANIZATION.docs.buildwithfern.com/<subpath>

The first `.docs.` cannot be changed. The second one can be whatever the customer wants the subpath to be (e.g., /docs)

Comment (dannysheridan) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:145]: @minaelee 

Talked to Andrew:

for custom subpath it is

- url: <subdomain>.docs.buildwithfern.com/<base_url>

We recommend folks to make the subdomain their organization name, but this isn't required.
When using a subpath, there is no `custom-domain` key used.

Comment (minaelee) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:145]: It seems like this info is better suited as updates to the `Custom domains` page (https://github.com/fern-api/fern/blob/main/fern/docs/pages/fern-docs/config/branding/custom-domain.mdx).

I can update there and swap out the info in both the docs-quickstarts with a link to the `Custom domains` page. How does that sound?

Comment (minaelee) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:145]: e.g.:

### Step 7: Set up a custom domain, subdomain, or subpath

If you wish to use a custom domain (such as `https://example.com`), subdomain (such as `https://docs.example.com`), or subpath (such as `https://example.docs.buildwithfern.com/learn`), visit the [Custom domain](/fern-docs/config/branding/custom-domain) page.
```

Comment (dannysheridan) [reply fern/docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx:145]: Love it! Let's do that. 

(I was mixed up in where I was leaving my comment.)

**Pull request merged**